src/main/webapp/DastFreeStyle.js

```diff
@@ -185,7 +181,7 @@ class DastFreeStyle {
     }
 
     commonScopeSettingVisibility(isVisible) {
-        let commonScopeRows = jq('.dastCommonScopeSetting');
```

Comment:
Is a new class created with this naming, dastCommonScopeSetting

If it is referred in one place, can we avoid the string concatenation here and call it directly as it was earlier.

Reply:
as its a variable. Appending Jquery identifer to get control details.